Library/ViewModels/LiveStreamCountdownViewModel.swift

```diff
@@ -152,6 +152,25 @@ LiveStreamCountdownViewModelInputs, LiveStreamCountdownViewModelOutputs {
                                                                     liveStream: liveStream,
                                                                     refTag: refTag)
     }
+
+    let startEndDates = Signal.zip(
+      configData.map { _ in AppEnvironment.current.scheduler.currentDate },
+      self.closeButtonTappedProperty.signal
+        .map { _ in AppEnvironment.current.scheduler.currentDate }
+    )
+
+    Signal.combineLatest(configData, startEndDates)
+      .takeWhen(self.closeButtonTappedProperty.signal)
+      .map(unpack)
+      .map { (configData, startDate, endDate) in
+        (configData.0, configData.1, configData.2, startDate, endDate) }
+      .observeValues { (project, liveStream, refTag, startDate, endDate) in
+        AppEnvironment.current.koala.trackClosedLiveStream(project: project,
```

Comment:
you could do the destructuring in the `observeValues` so that it's easier to read:

```swift
Signal.combineLatest(configData, startEndDates)
  .takeWhen(self.closeButtonTappedProperty.signal)
  .observeValues { configData, startAndEndDate in
    let (project, liveStream, refTag) = configData
    let (startDate, endDate) = startAndEndDate
```

or something like that.

or at the least put the trailing `}` on line 166 on a new line as it's our preferred way of handling multi line closures (we should add to our style guide).